.github/actions/spelling/expect.txt

```diff
@@ -794,6 +794,7 @@ userpreference
 UTCTIME
 vcenter
 vcpus
+vcruntime
```

Comment:
This change is no longer necessary. Remove if there are any other changes to be made.

Reply:
Removed